### ISSUE TITLE
XBox Controller Support

### ISSUE THREAD

**Issue by theodor349** [daemon3000/InputManager#43]
I have tried to set up an XBox controller. And it seems that most axis except for X, Y and the 3rd does not work.

**Comment by daemon3000** [comment]
What type of XBox controller? XBox 360 or XBox One?
Is the controller wired or wireless?
What Unity version are you using?
What platform do you have set as active in the build settings?
Did you try out example 05 in the plugin project? Does the controller work there?

**Comment by theodor349** [comment]
XBox 360
Wireless 
Unity version: 2018.2.16f1
OS: Windows 10
Platform: PC, Mac & Linux Standalone
---
Okay, I tried your example and it works. However, I tried setting the same up in my project and it does only work for the keyboard button.

Your project:
![capture](https://user-images.githubusercontent.com/14977008/47748990-62727300-dc8c-11e8-8014-2151d8b8abb8.PNG)
![capture](https://user-images.githubusercontent.com/14977008/47749036-7e761480-dc8c-11e8-8b32-c1e9e9e98c27.PNG)

My Project:
![capture](https://user-images.githubusercontent.com/14977008/47749062-951c6b80-dc8c-11e8-9711-937430c9957f.PNG)
![capture](https://user-images.githubusercontent.com/14977008/47749137-bbdaa200-dc8c-11e8-93e3-a17d4c774680.PNG)

**Comment by daemon3000** [comment]
Did you add the "ENABLE_X_INPUT" scripting symbol in the player settings? You should have gotten some warnings about it.

![](https://i.imgur.com/Gxm3muB.png)

**Comment by theodor349** [comment]
No, I had not added it. But now that I have I get the warning "No gamepad adapter has been assigned" and I can't figure out where I set that up.

**Comment by daemon3000** [comment]
Add the **XInputDotNetAdapter** component to the Input Manager game-object.
